research/format.markdown

```diff
@@ -0,0 +1,124 @@
+# Research on `format` keyword 
+ Changes in `format` Keyword Behaviour Across all JSON Schema Drafts: Additions, Deletions, and Upgrades.
+
+## Draft 2 to Draft 3
```

Comment:
I don't remember from the top of my head, but if `format` was not around before Draft 2, then it might be worth clarifying that. The intro can explicitly say something like "The `format` keyword was NOT available in Draft 0 and Draft 1"?

Reply:
`format` keyword is present in draft 1 and draf0. But there were no changes.
Then ig i'll mention Draft 1 to Draft2: and say No Changes

Reply:
Yep, makes sense. Let's be explicit to avoid confusion